table/column.go

```diff
@@ -82,6 +82,10 @@ func ToColumn(col *model.ColumnInfo) *Column {
 func FindCols(cols []*Column, names []string, pkIsHandle bool) ([]*Column, error) {
 	var rcols []*Column
 	for _, name := range names {
+		if name == "" {
+			rcols = append(rcols, nil)
```

Comment:
where to handle this nil? and do we need to add more nil-check in other dmls?

Reply:
`" "` is introduced by https://github.com/pingcap/tidb/pull/7707/files#diff-badfcd30d7596a08cd207b8e6ae778e6R553 stand the postion of `@xx-var`
and `nil` will never be handle, `nil`'s usage in here is a place holder and help other columns's offset be right(so later eval can works just like on-duplicate does)